src/Microsoft.Azure.CosmosRepository/Builders/PatchOperationBuilder.cs

```diff
@@ -32,24 +44,14 @@ public IPatchOperationBuilder<TItem> Replace<TValue>(Expression<Func<TItem, TVal
             return this;
         }
 
-        private string GetPropertyToReplace(PropertyInfo propertyInfo)
+        private string GetPropertyToReplace(MemberInfo propertyInfo)
         {
-            object[] attributes = propertyInfo.GetCustomAttributes(true);
-
-            if (attributes.Any() is false)
-            {
-                return _namingStrategy.GetPropertyName(propertyInfo.Name, false);
-            }
-
-            foreach (object attribute in attributes)
-            {
-                if (attribute is JsonPropertyAttribute jsonAttribute)
-                {
-                    return jsonAttribute.PropertyName;
-                }
-            }
-
-            return _namingStrategy.GetPropertyName(propertyInfo.Name, false);
+            JsonPropertyAttribute[] attributes =
+                propertyInfo.GetCustomAttributes<JsonPropertyAttribute>(true).ToArray();
+
+            return attributes.Length == 0
+                ? _namingStrategy.GetPropertyName(propertyInfo.Name, false)
+                : attributes.First().PropertyName;
```

Comment:
```suggestion
                : attributes[0].PropertyName;
```

It's an array, use the indexer.